framework/generated/khronos_generators/vulkan_generators/encode_pnext_struct_generator.py

```diff
@@ -162,6 +159,19 @@ def beginFile(self, gen_opts):
 
     def endFile(self):
         """Method override."""
+        for struct in self.all_extended_structs:
```

Comment:
I notice that we aren't filtering on the blacklist here, above, and in the previous implementation; but maybe no Next/pNext structs would be filtered out... hmmm..

Reply:
I'm following existing behavior to generate the same results.  Maybe something we can tweak in the future.